static/app/views/dashboardsV2/widgetBuilder/widgetBuilder.tsx

```diff
@@ -94,25 +96,31 @@ const DATASET_CHOICES: [DataSet, string][] = [
   // [DataSet.METRICS, t('Metrics (Release Health)')],
 ];
 
-const QUERIES = {
+const QUERIES: Record<DataSet, WidgetQuery> = {
   [DataSet.EVENTS]: {
     name: '',
     fields: ['count()'],
+    columns: [],
+    aggregates: ['count()'],
```

Comment:
Should we be updating the prebuilt widgets data with these fields as well?

Reply:
yup, going to update the widget library and default widget templates in a follow up